docs/content/sectigo/_index.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Sectigo"
+date: 2020-12-24T07:58:37-05:00
+lastmod: 2020-12-24T07:58:37-05:00
+description: "Directory Service interactions with the Sectigo CA API"
+weight: 10
+---
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/trisacrypto/testnet/pkg/sectigo.svg)](https://pkg.go.dev/github.com/trisacrypto/testnet/pkg/sectigo)
+
+
+The TRISA Directory Service issues certificates using the Sectigo Certificate Authority via its IoT Portal. Because the directory service must collect public key material in order to facilitate an initial trusted handshake for mTLS, it uses the Sectigo IoT Manager API as part of the VASP registration and verification process. The `github.com/trisacrypto/testnet/pkg/sectigo` package is a Go library for interacting with the API, implementing the endpoints and methods required by the directory service. The TestNet also provides a command line utility for interacting with the API for administrative and debugging purposes. This documentation describes the command line utility, which also gives an overview of how to use the API directly to issue and revoke certificates.
```

Comment:
I'll create an issue to update these Sectigo documents to point to the `trisacrypto/directory/pkg/sectigo` location.